modules/monitoring-creating-alert-routing-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-alert-routing-for-user-defined-projects_{context}"]
+= Creating alert routing for user-defined projects
+ 
+[IMPORTANT]
+====
+[subs="attributes+"]
+Alert routing for user-defined projects is a Technology Preview feature only. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview[https://access.redhat.com/support/offerings/techpreview].
+====
+
+You can create alerting routing for user-defined projects. 
+
+.Prerequisites
+
+* An administrator has enabled monitoring for user-defined projects.
+* You are logged in as a user that has the `alert-routing-edit` role for the project where you want to create alert routing.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a YAML file for alert routing. In this example, it is called `example-app-alert-routing.yaml`.
+
+. Add an `AlertmanagerConfig` YAML definition to the file. For example:
++
+[source,yaml]
+----
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: AlertmanagerConfig
+metadata:
+  name: example-routing
+  namespace: ns1
+spec:
+  route:
+    receiver: default
+    groupBy: [job]
+  receivers:
+  - name: default
+    webhookConfigs:
+    - url: https://httpbin.org/post
+----
++
+. Apply the resource to the cluster:
++
+[source,terminal]
+----
+$ oc apply -f example-app-alert-routing.yaml
+----
++
+It takes some time for the configuration to be propagated to the Alertmanager pods.
```

Comment:
I think this is totally fine as is. If you wanted, you could change to a simpler verb. Do any of the following make sense? 

`It takes some time for the configuration to be populated in the Alertmanager pods.`

`It takes some time for the configuration to be processed in the Alertmanager pods.`

Again, the original is straightforward and good. Simplifying the verb is only if you think some users might get stumped for a second on "propagated."

Reply:
"Propagated" and "processed" mean to slightly different things, but I agree that propagated is not as simple as it could be, so I'm going to use "applied" instead. Thanks, Jake!